lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/driver/YarnContainerManager.java

```diff
@@ -174,52 +193,81 @@ public float getProgress() {
     }
   }
 
+  /**
+   * RM Callback: RM reports an error.
+   * @param throwable An exception thrown from RM.
+   */
   @Override
   public void onError(final Throwable throwable) {
-    onRuntimeError(throwable);
+    this.onRuntimeError(throwable);
   }
 
+  /**
+   * NM Callback: NM reports start of a container.
+   * @param containerId ID of a newly started container.
+   * @param stringByteBufferMap currently not used.
```

Comment:
Have no idea - we need it because this method comes from YARN NM Callback interface.